tests/dialects/test_memref.py

```diff
@@ -31,7 +31,7 @@ def test_memref_load_i32():
     load = Load.get(memref_ssa_value, [])
 
     assert load.memref is memref_ssa_value
-    assert load.indices == []
+    assert load.indices == ()
```

Comment:
I must say, I am a bit surprised about how little changes were made in this file. Does this come from us not having tests where we pass non-empty lists or is the source code just the same? 
I hope its the first case, the second one would confuse me quite a bit :sweat_smile:

Reply:
Or I guess the third case could be that the source code to access tuples is the same as the one for lists and we always check each element individually.

Reply:
Yeah the subscripting api is the same